components/MastHead.vue

```diff
@@ -1,6 +1,9 @@
 <script setup lang="ts">
+	import type { Customer } from '~/types'
+
 	const { signOut } = useAuth()
-	const { state } = useCustomerStore()
+	const { getMe } = useCustomerStore()
```

Comment:
You shouldn't need to do `const { getMe } = useCustomerStore()` again. We're doing this already in `startApp.ts`, and then storing whatever it returns inside of the CustomerStore. 

You should be able to do 
```js
const  {state: customerState}  = useCustomerStore()

<template>
  <p>{{ customerState.firstName }}</p>
  <p>{{ customerState.email }}</p>
</template>
```